life_coach/helpers.py

```
@@ -0,0 +1,44 @@
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict
+import pytz
```

Ruff (0.12.2)

`pytz` imported but unused

Remove unused import: `pytz`

(F401)